### PR TITLE
ROE-2348 Set up-to-date config values for CiDev

### PR DESCRIPTION
**Pull request by mwestacott**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-2348

### Change description

* Changes, ready for when Node 18/ECS code is LIVE
* They now match current Mesos and chs-configs settings for CiDev

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -5,23 +5,26 @@ log_level = "debug"
 chs_url = "https://cidev.aws.chdev.org"
 piwik_url = "https://matomo.platform.aws.chdev.org/"
 piwik_site_id = "7"
+piwik_start_goal_id = "20"
+piwik_update_start_goal_id = "23"
 cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
 cookie_domain = "cidev.aws.chdev.org"
 account_local_url = "https://account.cidev.aws.chdev.org"
-piwik_start_goal_id = "20"
```
Comment (mwestacott) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:11]: Just rearranged where these values appear. Makes things a bit clearer and more closely matches how they were laid out in chs-configs.

**Comment by mwestacott** [comment]
Made this a draft review, just so that it doesn't get accidentally merged early.

**Review by markpit — APPROVED** [review]

**Comment by bantao-ch** [comment]
@AndreKramer7654 -- are we ready to turn ON the update trusts flag ?

**Comment by mwestacott** [comment]
> @AndreKramer7654 -- are we ready to turn ON the update trusts flag ?

So this won't be merged in until after the new pipeline and Node 18 code is LIVE, but after that, CiDev config should match what's currently in Mesos and there, FEATURE_FLAG_ENABLE_UPDATE_TRUSTS_30062023 is set to true. These are only CiDev settings...

Does that sound correct to you @AndreKramer7654 ?

**Review by AndreKramer7654 — APPROVED** [review]

**Review by dlloyd1-ch — APPROVED** [review]

**Pull request merged**